plugins/woocommerce/tests/php/src/Internal/Admin/EmailPreview/EmailPreviewTest.php

```diff
@@ -238,4 +239,45 @@ public function test_transient_values_in_preview() {
 
 		update_option( EmailPreview::get_email_style_settings_ids()[0], $original_value );
 	}
+
+	/**
+	 * Test that transient values are applied in subject
+	 */
+	public function test_transient_values_in_subject() {
+		$email_id = EmailPreview::DEFAULT_EMAIL_ID;
+		$key      = "woocommerce_${email_id}_subject";
+
+		$this->sut->set_email_type( EmailPreview::DEFAULT_EMAIL_TYPE );
+		$this->assertEquals( $this->sut->get_subject(), 'Your ' . self::SITE_TITLE . ' order has been received!' );
+
+		set_transient( $key, 'transient_subject', HOUR_IN_SECONDS );
```

Comment:
Should we clear transient values after test? This one only affects the `EmailPreview` class, but still might be a good thing to do.

Reply:
Fixed in 329a0af.